IMAGE_HEIGHT: 480
IMAGE_WIDTH: 640
